commit 8902a13e0df74900fc4dba202d34d61259eefae1
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2019-07-02T16:41:26+02:00 (3 months ago)
    
    TRITON-1752 Node triton documentation for --disks with flexible disk at instance creation

diff --git a/lib/do_instance/do_create.js b/lib/do_instance/do_create.js
index 9c681c6..d411591 100644
--- a/lib/do_instance/do_create.js
+++ b/lib/do_instance/do_create.js
@@ -548,13 +548,14 @@ do_create.options = [
         type: 'arrayOfString',
         helpArg: 'DATA',
         help: 'Configure disks for an instance with flexible disks. DATA is ' +
-            'a JSON object or "@FILE" to have disks loaded from FILE. Each ' +
-            'disk argument provided should have the format ' +
-            '\'{"size": POSITIVE INTEGER}\'. For the first disk (image disk)' +
-            ' it is recommended to provide the value \'{}\' which will ' +
-            'automatically set the size to the value specified into the ' +
-            'image used to create the instance. For the last disk, the ' +
-            'value "remaining" is also allowed.'
+            'a JSON object or "@FILE" to have disks loaded from FILE. ' +
+            '\'--disk=<JSON object>\' can be used more than once to provide ' +
+            'the required number of disks. Each JSON disk argument provided ' +
+            'should have the format \'{"size": POSITIVE INTEGER}\'. For the ' +
+            'first disk (image disk) it is recommended to provide the ' +
+            'value \'{}\' which will automatically set the size to the value ' +
+            'specified into the image used to create the instance. For the ' +
+            'last disk, the value "remaining" is also allowed.'
     },
 
     {
